### PR TITLE
[16.0][MIG] contract_variable_qty_timesheet: Migration to 16.0

### PR DESCRIPTION
**Pull request by danypr92**
This PR migrates the contract_variable_qty_timesheet module from Odoo version 15.0 to version 16.0.

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Thanks for the contribution.

Please preserve commit history following technical method explained in https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-16.0.

If the jump is between several versions, you have to modify the source branch in the main command to accommodate it to this circumstance.

**Comment by danypr92** [comment]
@pedrobaeza Done!

Thanks for the comment. I've followed the guide, and since it's only a one major version upgrade, I believe I've completed all the necessary tasks.

I realize that I haven't implemented the distribution to separate the timesheet entries in the analytic account based on the selected percentages. However, we need this simpler version first, and then we can discuss the distribution. Is that okay with you?

**Comment by pedrobaeza** [comment]
Ok about the procedure. Now reviewers should say the rest.

**Comment by danypr92** [comment]
Ping @carlosdauden @carolinafernandez-tecnativa please, can you review the proposal?

**Comment by pedrobaeza** [comment]
We (Tecnativa) are migrating this module to v17, so please review that one, and if similar/equal, then both PRs can be merged together.

File: contract_variable_qty_timesheet/readme/USAGE.rst [hunk 1]
```diff
@@ -0,0 +1,8 @@
+To use this module, you need to:
+
+#. Go to Invoicing > Sales > Contracts and select or create a new contract.
```
Comment (carolinafernandez-tecnativa) [inline contract_variable_qty_timesheet/readme/USAGE.rst]: It should be Go to Invoicing > Customers > Customer Contracts

**Review by carolinafernandez-tecnativa — COMMENTED** [review]
Functional review LGTM, could you please squash administrative commits and take a look of comment before?

Thanks.

**Review by carolinafernandez-tecnativa — APPROVED** [review]
LGTM Thanks :)

File: contract_variable_qty_timesheet/readme/USAGE.rst (hunk 1 above)
Comment (danypr92) [reply contract_variable_qty_timesheet/readme/USAGE.rst]: This was solved in the last update.

**Comment by pedrobaeza** [comment]
@danypr92 please review #1104

**Review by juanjosesegui-tecnativa — APPROVED** [review]

**Pull request merged**